vite.config.mts

```diff
@@ -0,0 +1,28 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+import { resolve } from 'path';
+
+export default defineConfig({
+    plugins: [react()],
```

Comment:
This project is using `React` ? I think you can delete the plugin and the `rollup` options

Reply:
After discussion it use React